data/src/main/java/com/linkedin/data/schema/SchemaToPdlEncoder.java

```diff
@@ -107,15 +110,41 @@ PdlBuilder newBuilderInstance(Writer writer)
   private String _namespace = "";
   private String _package = "";
 
+  private final boolean _trackWriteLocations;
+
+  private final Map<Object, PdlSchemaParser.ParseLocation> _writeLocations;
+
   /**
    * Construct a .pdl source code encoder.
    *
    * @param out provides the encoded .pdl destination.
    */
   public SchemaToPdlEncoder(Writer out)
   {
-    _writer = out;
+    this(out, false);
+  }
+
+  /**
+   * Construct a .pdl source code encoder.
+   *
+   * @param out provides the encoded .pdl destination.
+   * @param returnContextLocations Enable recording the context locations of schema elements during parsing. The
```

Comment:
I see why that would make sense but I used this naming because it matches the terminology in PdlSchemaParser [here](https://github.com/linkedin/rest.li/blob/b8fadffc4b9d7369ef0da31dd93fab4a527665db/data/src/main/java/com/linkedin/data/schema/grammar/PdlSchemaParser.java#L125)